package.json

```diff
@@ -101,6 +101,7 @@
     "newrelic": "^4.11.0",
     "node-fetch": "^3.2.10",
     "nx": "^15.0.9",
+    "phantom": "^6.3.0",
```

Comment:
Are some of the things added in this PR unintentional?

Reply:
Ill need to update from main.